test/unit/io.js

```diff
@@ -861,6 +861,70 @@ describe('Input/output', function () {
     });
   });
 
+  it('TIFF lzw compression shrinks test file', function (done) {
+    const startSize = fs.statSync(fixtures.inputUncompressedTiff).size;
```

Comment:
That's alright, sorry I got frustrated as well. It's my second ever PR and I just wasn't thinking about the courtesy and notifications you were probably getting. I'll try to get the tests working and then push it. Thanks for the great lib.